tests/src/test/scala-3/pureconfig/EnumerationConvertDerivationSuite.scala

```diff
@@ -0,0 +1,101 @@
+package pureconfig
+
+import scala.compiletime.testing.{typeCheckErrors, typeChecks}
+import scala.deriving.Mirror
+import scala.language.higherKinds
+
+import com.typesafe.config.{ConfigFactory, ConfigValueFactory, ConfigValueType}
+
+import pureconfig._
+import pureconfig.error.{CannotConvert, WrongType}
+import pureconfig.generic.derivation._
+
+class EnumerationConvertDerivationSuite extends BaseSuite {
```

Comment:
nit, just to use the same terminology as library code.

```suggestion
class EnumConvertDerivationSuite extends BaseSuite {
```